code/game/objects/effects/traps.dm

```diff
@@ -174,7 +174,11 @@ Add those other swinging traps you mentioned above!
 	. = ..()
 	if(AM.is_incorporeal())
 		return
-	if (istype(AM, /mob/living))
+
+	if(!tripped)
```

Comment:
unless tripped is labeled wrong, this means the damage is only applied when the trap has been tripped already

Reply:
Tripped means the trap is indeed triggered. Traps are triggered by pressure plates. Damage shouldn't be applied unless the trap is activated by a pressure plate, ergo, if the trap isn't active, it shouldn't do damage.